src/abbrev.rs

```diff
@@ -157,6 +157,7 @@ pub struct Abbreviation {
     tag: constants::DwTag,
     has_children: constants::DwChildren,
     attributes: Vec<AttributeSpecification>,
+    implicit_consts: Vec<i64>,
```

Comment:
I feel that we should add the implicit constant to the `AttributeSpecification` instead. It'll simply things a lot, and I don't think the increased memory usage will be significant (it may even be less memory for abbreviations with only a couple of attributes).

Reply:
I had assumed that an AttributeSpecification was 4 bytes but it looks like it's actually 16 (why are DwAt and DwForm u64s and not u16s?) so we'd break even after 3 attributes.  I could go either way here.

Reply:
DwAt and DwForm are u64s because that's what `read_uleb128` returns, and the standard doesn't explicitly define otherwise. Probably not a good reason, and I had noticed it in the past but never got around to doing anything about it. So this is something we may change in future.

This is a 2-3% performance hit to `bench_parsing_debug_info` (although I'm comparing to before this change, not the alternative implementation). So due to that, and due to the code complexity, I prefer changing this, but not strongly enough to require it. FWIW, gdb and llvm both store it in the attribute specification.

Reply:
On my system this actually seems to *speed up* `bench_parsing_debug_info`, not that that makes a ton of sense.

I don't feel very strongly about this so I'll change it.